src/layout/Main/index.tsx

```diff
@@ -137,21 +144,51 @@ function MainLayout({ children }: PropsWithChildren) {
   const globalElements = useAtomValue(globalElementsAtom);
   const { walletAddress } = useConnectedWallet();
   const connectWalletModal = useConnectWalletModal();
-  const [searchParam, setSearchParam] = useSearchParamState<string>(
-    CHAIN_NAME_SEARCH_PARAM,
-    DefaultChainName,
-  );
+  const [searchParams, setSearchParams] = useSearchParams();
 
-  const name = useMemo(() => {
-    if (wallet.status === WalletStatus.WALLET_CONNECTED) {
-      return wallet.network.name === "testnet" ? "xplatestnet" : "xpla";
-    }
-    return searchParam ?? DefaultChainName;
-  }, [searchParam, wallet.network.name, wallet.status]);
+  const handleModalClose = useCallback(() => {
+    const newParams = new URLSearchParams(searchParams);
+    newParams.set(CHAIN_NAME_SEARCH_PARAM, DefaultChainName);
+    setSearchParams(newParams);
+  }, [searchParams, setSearchParams]);
+
+  const { isOpen, open } = useInvalidPathModal({
```

Comment:
```suggestion
  const { open } = useInvalidPathModal({
```

please cleanup unused variables.

Reply:
[b384f98](https://github.com/dezswap/dezswap-web-app/pull/248/commits/b384f98fdde68ac4fd6cd7e1f79a1f1113821fc5) is updated thank you